server/fixity.go

```diff
@@ -139,3 +155,118 @@ func (s *RESTServer) scanfixity() {
 	}
 	log.Println("Ending scanfixity. duration = ", time.Now().Sub(starttime))
 }
+
+func (s *RESTServer) GetFixityHandler(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+
+	item := r.FormValue("item")
+	start := r.FormValue("start")
+	end := r.FormValue("end")
+	status := r.FormValue("status")
+
+	log.Println("GetFixityHandler called")
+	log.Println("item= ", item)
+	log.Println("start= ", start)
+	log.Println("end= ", end)
+	log.Println("status= ", status)
+
+	startValue, startErr := startValidate(start)
```

Comment:
use `err` instead of `startErr`.

Reply:
and do the same for the other errors later in the function.

Reply:
I, personally, like seeing the errors fro the 400s - are those the one to which you are referring?

Reply:
uhh. I think this reply is to the wrong comment so I will reply as though it is with the comment I think it should have been attached to. Having no results is not an error, so why should it return a 400? I do think passing a malformed time should cause an 400 response, though. And when looking up a particular record, passing a non-existent `id` should return a 404.

Reply:
OK, I changed the no results to a 200, and nonexistent passes a 404.